Practice3/Practice3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Practice3
+{
+    class Task
```

Comment:
please, specify the access modifier for the class

Reply:
Done